lib/ansible/modules/cloud/openstack/os_port.py

```diff
@@ -60,11 +60,13 @@
                   - ip_address: ..."
    extra_dhcp_opts:
      description:
-        - "Extra dhcp options to be assigned to this port.  Extra options are
-          supported with dictionary structure.
+        - "Extra dhcp options to be assigned to this port. Extra options are
```

Comment:
nit: this is a leading quote, without an ending one.

Reply:
The ending quote is present on row 70, agreed it looks weird but I only added the Note line. Also the above option is in the same format.